gradeforge/utils.py

```diff
@@ -176,3 +174,12 @@ def section_link(semester, uid):
     https://ssb.onecarolina.sc.edu/BANP/bwckschd.p_disp_detail_sched?term_in=201808&crn_in=12566'''
     base_url = 'https://ssb.onecarolina.sc.edu/BANP/bwckschd.p_disp_detail_sched'
     return "%s?term_in=%s&crn_in=%s" % (base_url, semester, uid)
+
+
+def bookstore_link(semester, department, code, section):
+    '''Example:
+    https://ssb.onecarolina.sc.edu/BANP/bwckbook.site?p_term_in=201808&p_subj_in=WGST&p_crse_numb_in=621&p_seq_in=001'''
+    section = str(section).zfill(3)
+    base_url = 'https://ssb.onecarolina.sc.edu/BANP/bwckbook.site'
+    url_format = "%s?p_term_in=%s&p_subj_in=%s&p_crse_numb_in=%s&p_seq_in=%s"
+    return url_format % (base_url, semester, department, code, section)
```

Comment:
IIRC, `%` stile formatting is considered depricated, although still supported in 3.X, should probably use `str.format()` instead. Also, with `str.format` you can use hashtables as arguments as shown [here](https://stackoverflow.com/questions/5952344/how-do-i-format-a-string-using-a-dictionary-in-python-3-x). If you really wanted to, you could probably use `__locals__` as the table >:)

Reply:
I like this idea, but there's a *lot* of code currently using `%` for formatting. Maybe clean it up only if there's a separate relevant change being made.